core/lib/presentation/extensions/list_extensions.dart

```diff
@@ -16,4 +16,16 @@ extension ListExtensions<T> on List<T> {
       validBuilder.build().toList(),
       validBuilder.build().toList());
   }
+
+    int countOccurences(T value) {
+    int count = 0;
+
+    for (T element in this) {
+      if (element == value) {
+        count++;
+      }
+    }
+    
+    return count;
+  }
```

Comment:
why don't use `where`?

```suggestion
int countA = B.where((element) => element == A).length;
```

Reply:
Done